MobiFlight/MidiBoard/MidiBoardDefinition.cs

```diff
@@ -36,27 +37,32 @@ public class MidiBoardDefinition
 
         private Dictionary<string, string> inputNameToLabelDictionary;
 
+        [JsonIgnore]
```

Comment:
This needs to be added so when re-exporting the object to JSON for validation after migration the property isn't written out (it's not defined in the schema)

Reply:
What's the use case for exporting to JSON?

Reply:
It's done as part of the schema validation flow to support migrating old files to a new schema before validation. You'll see later on, there's code comments about it.